internal/ingress/annotations/loadbalancer/main.go

```diff
@@ -214,18 +216,26 @@ func parseCidrs(ing parser.AnnotationInterface) (out []string, err error) {
 	for _, inboundCidr := range cidrConfig {
 		ip, _, err := net.ParseCIDR(inboundCidr)
 		if err != nil {
-			return out, err
+			return v4CIDRs, v6CIDRs, err
 		}
 
 		if ip.To4() == nil {
-			return out, fmt.Errorf("CIDR must use an IPv4 address: %v", inboundCidr)
```

Comment:
ip.To4() is not enough, ip.To4() for "::/0" will return non-nil, and cause SG update fails.
Example checks in V2 branch: https://github.com/kubernetes-sigs/aws-alb-ingress-controller/blob/v2/pkg/build/lb_security_group.go#L145